### PR TITLE
128 update README.md for PowerShell execution policies

### PR DESCRIPTION
**Pull request by gdasher**
#  update README.md for PowerShell execution policies #

## 🗣 Description ##

This change updates README.md to provide more accurate instructions for invoking signed powershell artifats.

## 💭 Motivation and context ##

Dolphin includes code signing, which changes the instructions for running the module.

## 🧪 Testing ##

CodeSigning will be retested as part of release testing.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.

### CODE REVIEW

**Review by gdasher — COMMENTED** [review]
You can preview the new readme with formatting here: https://github.com/cisagov/ScubaGear/tree/128-update-readme

**Comment by gdasher** [comment]
Note that i didn't test every case (e.g. unrestricted on non-windows I haven't tested in a while)--I tried to copy from the Windows docs.

File: README.md [hunk 1]
```diff
@@ -28,15 +28,16 @@ To download ScubaGear:
...
-
 Then run:
 
 ```powershell
 .\Setup.ps1 #Installs the required modules
 Import-Module -Name .\PowerShell\ScubaGear #Imports the tool into your session
 ```
 
+> **Note**
+> Starting with release 0.3.0, ScubaGear is signed by a commonly trusted CA.  Depending on the [PowerShell execution policy](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-5.1) of the system running ScubaGear, different steps may be required before running running ScubaGear.  See [PowerShell Execution Policies](#powershell-execution-policies) for more details.
```
Comment (nanda-katikaneni) [inline README.md:39]: Typo in the text  - remove running from "...different steps may be required before running running ScubaGear".

**Review by nanda-katikaneni — COMMENTED** [review]
Except for a minor typo noted, looks good to me.

File: README.md (hunk 1 above)
Comment (gdasher) [reply README.md]: Good catch.  Done!

**Comment by gdasher** [comment]
@tkol2022 any comments?

**Review by tkol2022 — APPROVED** [review]
I tested the suggested execution policy using the command below against a zip file created with the signed Powershell files on a fresh virtual machine and it works as expected - no warnings or prompts that mention execution policy restrictions.

`Set-ExecutionPolicy RemoteSigned`

**Pull request merged**